app/javascript/controllers/auto_citation_controller.js

```diff
@@ -1,18 +1,21 @@
 import { Controller } from "@hotwired/stimulus";
 
+const PURL_PLACE_HOLDER = ":link will be inserted here automatically when available:"
+const DOI_PLACE_HOLDER = ":DOI will be inserted here automatically when available:"
+
 export default class extends Controller {
   static targets = ["titleField", "manual", "auto", "switch",
     "contributorFirst", "contributorLast", "contributorRole", "contributorOrg",
     "embargoYear", "embargo"];
 
   connect() {
-    this.purl = this.data.get("purl") || ":link will be inserted here automatically when available:" // Use a real purl on a persisted item or a placeholder
+    this.purl = this.data.get("purl") || PURL_PLACE_HOLDER // Use a real purl on a persisted item or a placeholder
     this.doi = this.data.get("doi") || ""
 
     this.updateDisplay()
 
-    // If the manualTarget matches the autoTarget or is blank, then display the auto.
-    const showAuto = this.manualTarget.value === '' || this.manualTarget.value === this.autoTarget.value
+    // If the manualTarget is blank or the autoTarget matches the citation, then display the auto.
```

Comment:
The bug here is that the manual will never match the auto because the auto has the PURL and DOI inserted while the manual has the place holders. I did not like using a long line here with 2 replaces in a single line so moved this into a method to hopefully make the logic a bit more clear.